src/fr/commencer.md

```diff
@@ -26,14 +26,16 @@ Par exemple, si votre clé API est
 * votre ISS (votre ID de service) est `26785a09-ab16-4eb0-8407-a37497a57506`
 * votre clé secrète est `3d844edf-8d35-48ac-975b-e847b4f122b0`
 
-Par conséquent, vous devez définir l’en-tête HTTP à la valeur suivante :
+::: tip NOUVEAU: Inclure la clé API entière en appellant l'API
```

Comment:
It might be only one `l`: `en appelant`. I like Google translate version better though:

```
Inclure la clé API entière lors de l'appel de l'API
```